src/server/app/shared/plugins/auth.js

```diff
@@ -21,3 +23,32 @@ export async function validateUser(req, username, password, h) {
     return new Error(err);
   }
 };
+
+export async function generateToken(auth, username, password) {
+  let user, group, secret, payload, token;
```

Comment:
These could all be constants within the function as well, so that we clarify that we don't expect the variables to mutate.